sktime/proba/base.py

```diff
@@ -158,6 +158,18 @@ def _method_error_msg(self, method="this method", severity="warn", fill_in=None)
         else:
             return msg
 
+    def _get_bc_params(self, to_broadcast, dtype=None):
```

Comment:
can I suggest two changes, as a thought? not blocking

* turning the arg `to_broadcast` into an *args, same how `np.broadcast_arrays` handles its input
* make the input optional (none default), the default gets all params from self except index, columns, via `self.get_params`

Reply:
> turning the arg to_broadcast into an *args

I can do but later on in the method, `self.index` and `self.columns` may be appended so `args` would need to be cast to a list anyway. Would you still prefer to use `*args`?

> make the input optional (none default).

Yes I will make the change.

Reply:
> I can do but later on in the method, self.index and self.columns may be appended so args would need to be cast to a list anyway. Would you still prefer to use *args?

Hm, we could just use tuples throughout: `np.broadcast_arrays` wants a tuple, and a `*to_broadcast` also is a tuple.

Reply:
regarding going back/forth on the signature - that's a normal part of design, and feel free to voice strong opinions! there's no clear right & wrong, only more qualitative criteria like coherence/coupling, ease of use, consistency with formalism orconceptual models, etc